config/system/alt.yaml

```diff
@@ -30,3 +30,12 @@ run_trigger:
   vlan_start: 1001
   vlan_end: 1009
   egress_vlan: 121
+
+mininet_subnet:
+  base_ip: 192.168.1.0
+  prefix_length: 24
+
+# Define external DHCP subnets to be used by test hosts
+external_dhcp_subnets:
```

Comment:
maybe just "external_subnets" -- I'm not sure "DHCP" matters... e.g. it applies to static IPs as well.